data/hai/unfettered 2 war preparations.txt

```diff
@@ -102,7 +167,6 @@ mission "<uhai> 10: Titanium"
 	description `Mine 40 tons of Titanium from <waypoints> for <payment>.`
 	source "Firelode"
 	waypoint "Ik'kara'ka"
-	destination "Darkcloak"
```

Comment:
Are you sure about this?

Reply:
yeah I want you to bring it directly to her on Firelode